src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -149,10 +158,32 @@ local function tabulateCommon( bPaged, ... )
 end
 
 function tabulate( ... )
+    tArgs = { ... }
+    for i=1,#tArgs,2 do
```

Comment:
My point was "let's make less calls", but it's not an issue while we remain on Lua 5.1 anyway.